### PR TITLE
Added script that cleans docker data

### PR DESCRIPTION
**Pull request by qeude**
## Description

Basic shell script that clean everything related to docker (images, containers, volumes).

## Type of change

- [x] New script command

## Screenshot

Screen Recording 2021-06-06 at 21 28 15

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/docker/clean-docker.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# @raycast.title Clean Docker
+# @raycast.mode compact
+# @raycast.packageName Docker
+
+# Optional parameters:
+# @raycast.icon 🧹
+# @raycast.needsConfirmation true
+
+# Documentation:
+# @raycast.description Script that cleans Docker images, volumes, and containers
```
Comment (unnamedd) [inline commands/developer-utils/docker/clean-docker.sh:14]: Don’t forget to add your credits here @qeude.

File: commands/developer-utils/docker/clean-docker.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Clean Docker
```
Comment (unnamedd) [inline commands/developer-utils/docker/clean-docker.sh:5]:
```suggestion
# @raycast.title Clean
```

File: commands/developer-utils/docker/clean-docker.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```
Comment (unnamedd) [inline commands/developer-utils/docker/clean-docker.sh:2]: You need to add the dependency information here and the check if docker is installed in the system or not. Check this [pull request ](https://github.com/raycast/script-commands/pull/459/files)to copy how it was added there.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @qeude, thanks for your contribution adding another Script Command to Docker. 
I made some small comments here, please address them and let us know for a second round of review!

**Comment by qeude** [comment]
Hi @unnamedd,
Thanks for your feedback, changes have been commited 🙌

**Review by unnamedd — APPROVED** [review]
Looks good to me! It's approved ✅ 

We're ready to go @mattisssa

**Pull request merged**